src/ui/src/configurables/base/theme-colors.ts

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2018- The Pixie Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// These aren't quite the Material UI colors, but they follow the same principles.
+export const COLORS = {
+  PRIMARY: {
+    '100': '#D0FBFB',
+    '200': '#A1F7F7',
+    '300': '#6DF3F3',
+    '400': '#35EEEE',
+    '500': { // Brand primary color!
+      dark: '#12D6D6',
+      light: '#12D6D6',
+    },
```

Comment:
**Nit (non-blocking)**: Instead of breaking convention here, how about a '500' and a '550' (then you can follow the normal pattern up in `mui-theme.tsx` as well)?

Reply:
MUI is new to me and I wasn't sure how to best structure this. Updated to reflect that :+1: